components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/pages/src/app/components/Apis/Details/NavBar.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react'
```

Comment:
Thanks @prasa7  for review, I have added missing licence headers with this [commit](https://github.com/wso2/carbon-apimgt/pull/4086/commits/b421388703ea2b66b11fdc59998fb0c614188d22)